src/linkml_map/transformer/transformer.py

```diff
@@ -157,22 +156,22 @@ def _get_class_derivation(self, target_class_name: str) -> ClassDerivation:
         ancmap = self._class_derivation_ancestors(cd)
         if ancmap:
             cd = deepcopy(cd)
-            for anc in ancmap.values():
-                for k, v in anc.__dict__.items():
+            for ancestor in ancmap.values():
+                for k, v in ancestor.__dict__.items():
```

Comment:
codespell was convinced this should be `and` ==> change it to `ancestor` to shut codespell up.

Reply:
I have made the same changes myself in other linkml repos :D

Reply:
I suppose at least it does enforce good variable naming... even if it is a bit of a nuisance!